tox.ini

```diff
@@ -0,0 +1,9 @@
+[tox]
+envlist=py27,py34,py35
+[testenv]
+deps=
+  pytest
+  pytest-xdist
+  requests
+  mock
```

Comment:
What are pytest-xdist and mock for?

Reply:
pytest-xdist is for running the tests in parallel (the -n 4 param in tox.ini). It is MUCH faster that standard serial running.
mock is redudant and I shouldn't put it here.